spec/factories/volunteer_factory.rb

```diff
@@ -2,6 +2,7 @@
   factory :volunteer do
     name { Faker::Name.name }
     email { Faker::Internet.email }
-    phone { Faker::PhoneNumber.phone_number }
+    phone { "1234567890" }
```

Comment:
Is there a reason for the change in this line?

Reply:
When trying to create a volunteer `FactoryBot.build(:volunteer)` we get an invalid phone number error. My guess is that it has something to do with this validation:
`  validates :phone, presence: true, format: /\A\+?[\d\(\)\-\ ]{7,}\z/
`

Reply:
Yeah, I'd be ok with removing the regex validation from phone numbers entirely, and just let people enter whatever they want. Just my 2 cents though :)